docusaurus.config.ts

```diff
@@ -172,6 +172,11 @@ const config: Config = {
             theme: prismThemes.github,
             darkTheme: prismThemes.dracula,
         },
+        algolia: {
+            appId: 'GEUAOEJ3NK',
+            apiKey: '329ccc709854898024bad32d341d441f',
```

Comment:
It's safe to commit this according to docs: https://docusaurus.io/docs/search#connecting-algolia